android/src/main/kotlin/com/frontegg/flutter/FronteggMethodCallHandler.kt

```diff
@@ -25,11 +26,22 @@ class FronteggMethodCallHandler(
         }
     }
 
-    private fun login(result: MethodChannel.Result) {
+    private fun login(call: MethodCall, result: MethodChannel.Result) {
+        val loginHint = call.argument<String>("loginHint")
+
         activityProvider.getActivity()?.let {
-            FronteggAuth.instance.login(it)
+            FronteggAuth.instance.login(
+                activity = it,
+                loginHint = loginHint,
+                callback = {
+                    result.success(null)
+                }
+            )
+        }
+        /// Callback never be called if not Embeded Mode
+        if (!FronteggApp.getInstance().isEmbeddedMode) {
+            result.success(null)
```

Comment:
you call result.success(null) immediately after triggering async calls in some cases. It could be better to call result.success(null) only within the callback, ensuring a consistent response pattern

Reply:
Fixed on Native part. Fixed here too